hudi-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -1598,7 +1597,7 @@ public void testAvroLogRecordReaderWithMixedInsertsCorruptsAndRollback(ExternalS
     scanner.close();
   }
 
-  @ParameterizedTest
+  /*@ParameterizedTest
```

Comment:
Why changing this one?

Reply:
I based of this patch on our release branch which had this test unintentionally pulled in. but guess w/ latest state of the branch, its not an issue. this test is not pulled in only